google/cloud/bigtable/table.py

```diff
@@ -1071,13 +1087,14 @@ def _do_mutate_retryable_rows(self):
         if self.timeout is not None:
             kwargs["timeout"] = timeout.ExponentialTimeout(deadline=self.timeout)
 
+        # todo confirm this change
         try:
             responses = data_client.mutate_rows(
-                self.table_name,
-                entries,
+                table_name=self.table_name,
+                entries=entries,
                 app_profile_id=self.app_profile_id,
                 retry=None,
-                **kwargs
+                # **kwargs
```

Comment:
not sure what do do with the `**kwargs` here

Reply:
leaving them in gives errors in the system tests (IIRC something related to too many arguments passed in)

Reply:
This likely will need investigating then. I can help if needed. But this would likely break something for users.

Reply:
Nevermind, just tried again and it worked :)